doc/source/whatsnew/v1.1.0.rst

```diff
@@ -701,6 +701,7 @@ Groupby/resample/rolling
 - Bug in :meth:`DataFrame.groupby` where a ``ValueError`` would be raised when grouping by a categorical column with read-only categories and ``sort=False`` (:issue:`33410`)
 - Bug in :meth:`GroupBy.first` and :meth:`GroupBy.last` where None is not preserved in object dtype (:issue:`32800`)
 - Bug in :meth:`Rolling.min` and :meth:`Rolling.max`: Growing memory usage after multiple calls when using a fixed window (:issue:`30726`)
+- Bug where subclasses were not preserved through groupby operations (:issue:`28330`)
```

Comment:
can you enumerate which operations here

Reply:
you can also do general classes of things, e.g. fitler, transform, agg if it affects lots of functions.